### PR TITLE
NF: replace `is`

### PR DESCRIPTION
**Pull request by Arthur-Milchior**

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt [hunk 1]
```diff
@@ -581,11 +581,11 @@ abstract class AbstractFlashcardViewer :
             .addPathHandler("/") { path: String ->
                 try {
                     val file = File(mediaDir, path)
-                    val `is` = FileInputStream(file)
+                    val equalTo = FileInputStream(file)
                     val mimeType = guessMimeType(path)
                     val headers = HashMap<String, String>()
                     headers["Access-Control-Allow-Origin"] = "*"
-                    val response = WebResourceResponse(mimeType, null, `is`)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:588]: I guess that here "is" is a abbreviation for InputStream

File: AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.kt [hunk 2]
```diff
@@ -502,8 +502,8 @@ internal class ZipFile(path: String?) {
     @Throws(IOException::class)
     fun writeStr(entry: String?, value: String) {
         // TODO: Does this work with abnormal characters?
-        val `is`: InputStream = ByteArrayInputStream(value.toByteArray())
-        val bis = BufferedInputStream(`is`, BUFFER_SIZE)
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/libanki/AnkiPackageExporter.kt:506]: ditto

File: AnkiDroid/src/main/java/com/ichi2/libanki/importer/Anki2Importer.kt [hunk 3]
```diff
@@ -821,16 +821,16 @@ open class Anki2Importer(col: Collection?, file: String) : Importer(col!!, file)
      * Return the contents of the given input stream, limited to Anki2Importer.MEDIAPICKLIMIT bytes This is only used
      * for comparison of media files with the limited resources of mobile devices
      */
-    @KotlinCleanup("rename `is` ")
-    private fun _mediaPick(`is`: BufferedInputStream): ByteArray? {
```
Comment (BrayanDSO) [inline AnkiDroid/src/main/java/com/ichi2/libanki/importer/Anki2Importer.kt:825]: ditto

**Review by BrayanDSO — CHANGES_REQUESTED** [review]
Some comments on the changes on `src/main`

I haven't looked to `src/test` because the changes should be fine as long as they pass CI (currently they don't)

**Review by david-allison — CHANGES_REQUESTED** [review]
Tests don't pass

File: AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt (hunk 1 above)
Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt:588]: Right, thanks

Comment (Arthur-Milchior) [reply AnkiDroid/src/main/java/com/ichi2/anki/AbstractFlashcardViewer.kt]: Thanks, you're perfectly right

**Review by BrayanDSO — APPROVED** [review]

**Review by david-allison — APPROVED** [review]

**Pull request merged**